docs/eventlisteners.md

```diff
@@ -632,6 +633,17 @@ spec:
     value: $(body.pull_request.head.short_sha)
 ```
 
+## Response from the EventListener
```

Comment:
Very small nit: Call this EventListener Response instead?

Reply:
Done.